### PR TITLE
Fix minor typos

### PR DESCRIPTION
**Pull request by cocoatomo**
These are just miscellaneous fixes.

### CODE REVIEW

**Comment by Mariatta** [comment]
Thanks :)

File: Doc/license.rst [hunk 1]
```diff
@@ -571,7 +571,7 @@ The :mod:`test_epoll` contains the following notice::
 Select kqueue
 -------------
 
-The :mod:`select` and contains the following notice for the kqueue interface::
+The :mod:`select` contains the following notice for the kqueue interface::
```
Comment (zware) [inline Doc/license.rst:574]: This still isn't quite right, it now reads `"The select contains the following ..."`, which is nonsensical.  Either remove the leading 'The', or (preferably) add 'module' after `select`.

Comment (Mariatta) [reply Doc/license.rst:574]: Sorry about that 😞 Fixed here: https://github.com/python/cpython/pull/1174

**Comment by terryjreedy** [comment]
Doc changes should be backported, at least to maintained 3.x versions.  Miss Islington makes this usually easier now ;-).  bpo-32664 is a report of the missing '|' in  Lib/test/exception_hierarchy.txt.  (Not an issue in 2.7.)  This should be fixed in 3.6.

I could just add the 3.6 backport tag, but then there would be the incorrect fix to Doc/license.rst.  That could then be fixed by tagging #1174, but I don't know if you think this the best thing to do.

**Comment by Mariatta** [comment]
Hmm, ok I think we can backport both: this PR first, then followed by  backport of GH-1174

**Pull request merged**